<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0">
    <widget class="QDialogButtonBox" name="OKCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="ChangeMain">
     <property name="text">
      <string>Change Main</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="CheckBox_QuitConfirmation">
     <property name="text">
      <string>Ask confirmation before quitting</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="EngRadioButton">
       <property name="text">
        <string>English</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ItaRadioButton">
       <property name="text">
        <string>Italiano</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="SettingsLabel">
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="CheckBox_Music">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Music</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Slider_Music">
       <property name="maximumSize">
        <size>
         <width>184</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>42</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="MusicSpinBox">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>42</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="LanguageLabel">
     <property name="text">
      <string>Language - Lingua:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBox_sfx">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Sound effects</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OKCancel</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OKCancel</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_Music</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MusicSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MusicSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slider_Music</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
